Integrations/SecurityAdvisor/SecurityAdvisor_test.py

```diff
@@ -0,0 +1,41 @@
+import SecurityAdvisor
+import unittest
+from mock import patch, Mock
+
+URL_SUFFIX = 'apis/coachuser/'
```

Comment:
we use pytest to run our tests.
So please rewrite the unit tests to fit pytest format

Reply:
This is done.